packages/builder/src/components/design/settings/controls/ColumnEditor/BasicColumnEditor.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
```

Comment:
Could this entire file not just be a tiny wrapper around the existing `ColumnEditor` that simply passes on down the value of a new `allowCellEditing` prop?

e.g.
- Add `allowCellEditing` prop to `ColumnEditor`, defaulting to true
- `allowCellEditing` is always passed down from `ColumnEditor` to `ColumnDrawer`
- New `BasicColumnEditor` is a tiny component that simply passes down `{ ...$$props }` and `allowCellEditing={false}`

This would then mean all this duplication could be removed.

The only other difference I can see is the text: "table" => "dynamic filter", so we could just add another optional prop to the `ColumnEditor` to allow customising this text. In my opinion that's still much better than duplicating the file. 

Were there any other differences between the files that I'm missing?

Reply:
Good call; removed duplication!